drivers/bluetooth/hci/CMakeLists.txt

```diff
@@ -1,11 +1,24 @@
 # SPDX-License-Identifier: Apache-2.0
 
+# Remove after 3.7.0 is released
+if(CONFIG_BT_RPMSG)
+	message(FATAL_ERROR  "CONFIG_BT_RPMSG has been renamed to CONFIG_BT_HCI_IPC")
+endif()
```

Comment:
These should be moved to a .c file ideally, otherwise a project cannot be configured and menuconfig cannot be ran to fix this issue

Reply:
I had them in a C file originally but I chose to move them here because the error message layout was much clearer for the user.